.github/workflows/test_ids_check.yml

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2024 3mdeb <contact@3mdeb.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Test IDs check
+
+# Only try to run if a robot file or the test cases mapping file were modified
+on:
+  pull_request:
+    paths:
+      - '**.robot'
+      - 'test_cases.json'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository and submodules
+      uses: actions/checkout@v4
+      with:
+        submodules: 'recursive'
```

Comment:
Not sure how much time it would actually save, but for this check submodules aren't used.